bundles/org.openhab.binding.oppo/README.md

```diff
@@ -0,0 +1,183 @@
+* In order for the direct IP connection to work while the player is turned off, the Standby Mode setting must be set to "Quick Start" in the Device Setup menu.
+* Likewise if the player is turned off, it may not be discoverable by the Binding's discovery scan.
+* If you experience any issues using the binding, first ensure that the player's firmware is up to date with the latest available version (especially on the older models). 
+* For the older models, some of the features in the control API were added after the players were shipped.
+* On Linux, you may get an error stating the serial port cannot be opened when the Oppo binding tries to load.
+* You can get around this by adding the `openhab` user to the `dialout` group like this: `usermod -a -G dialout openhab`.
+* Also on Linux you may have issues with the USB if using two serial USB devices e.g. Oppo and RFXcom.
+* See the [general documentation about serial port configuration](/docs/administration/serial.html) for more on symlinking the USB ports.
+* Here is an example of ser2net.conf you can use to share your serial port /dev/ttyUSB0 on IP port 4444 using [ser2net Linux tool](https://sourceforge.net/projects/ser2net/) (take care, the baud rate is specific to the Oppo player):
+
+```
+4444:raw:0:/dev/ttyUSB0:9600 8DATABITS NONE 1STOPBIT LOCAL
+```
+
+## Channels
+
+The following channels are available:
+
+| Channel ID        | Item Type | Description                                                                                                    |
+|-------------------|-----------|----------------------------------------------------------------------------------------------------------------|
+| power             | Switch    | Turn the power for the player on or off                                                                        |
+| volume            | Dimmer    | Control the volume for the player (0-100%)                                                                     |
+| mute              | Switch    | Mute or unmute the volume on the player                                                                        |
+| source            | Number    | Select the source input for the player (number of available options varies by model)                           |
+| play_mode         | String    | Indicates the current playback mode of the player (ReadOnly)                                                   |
+| control           | Player    | Simulate pressing the transport control buttons on the remote control (play/pause/next/previous/rew/ffwd)      |
+| time_mode         | String    | Sets the time information display mode on the player (T, X, C, K)                                              |
+| time_display      | String    | The playback time display (same as on the player's LCD screen) (ReadOnly)                                      |
```

Comment:
I changed it. But there is also value in displaying the time as it appears on the LCD screen. So I added another channel 'time_display_raw' to display it in the original format.

Reply:
Can't you set the format in the XML file as it is displayed on the LCD?

Reply:
@fwolter 
The string is '00:00:00' (hour/minute second). Would channel have to be a DateTime and use the pattern "%1$tH:%1$tM:%1$tS"? That was the best example that I could find.

Reply:
That looks good. The Channel could be fed by something like this `new QuantityType<>(timeDisplay, SmartHomeUnits.SECOND)`.

Reply:
I couldn't get it to format properly. So I removed the extra channel and added javascript to do the transformation to the README in case the user wants it.

Reply:
Sorry for recommending this. It's not possible as you figured out yourself.